src/hooks.tsx

```diff
@@ -369,6 +372,35 @@ const sortDataInOrder = <T extends DataType>(
   });
 };
 
+const sortByColumn = <T extends DataType>(
+  data: RowType<T>[],
+  sortColumn: string,
+  columns: ColumnStateType<T>[]
+): RowType<T>[] => {
+  let isAscending = null;
+  let sortedRows: RowType<T>[] = [...data];
+  // loop through all columns and set the sort parameter to off unless
+  // it's the specified column (only one column at a time for )
+  columns.map(column => {
+    // if the row was found
+    if (sortColumn === column.name) {
+      isAscending = !column.sorted.asc;
```

Comment:
Why the `!` here? Doesn't that set `isAscending` to the opposite of what we want?

Reply:
Good catch.  This actually does do what we expect but it reads weird because of the way the TOGGLE_SORT action works (basically it toggles the state after sorting instead of before).  I think I'll make a PR to make this more straightforward.